kura/org.eclipse.kura.core.keystore/META-INF/MANIFEST.MF

```diff
@@ -38,7 +38,7 @@ Import-Package: com.eclipsesource.json;version="0.9.5",
  org.eclipse.kura.marshalling;version="[1.0,2.0)",
  org.eclipse.kura.message;version="[1.4,2.0)",
  org.eclipse.kura.rest.utils;version="[1.0,2.0)",
- org.eclipse.kura.security.keystore;version="[1.0,2.0)",
+ org.eclipse.kura.security.keystore;version="[1.1,2.0)",
```

Comment:
Not sure why this was [1.0,2.0) since it is implementing the API. It should be [1.1,1.2), right?